AndelaOSP/art-dashboard
# #160140506 Add category and location cards

## What does this PR do?

Changes display of Andela centers and categories from table to cards

## Description of Task to be completed?

```
Display categories and locations on a card
```

## How should this be manually tested?

* Log in
* Click on either the `Andela Centres` or `Asset Categories` links to see the changes

## What are the relevant pivotal tracker stories?

[160140506](https://www.pivotaltracker.com/story/show/160140506)

## Any background context you want to add?

N/A

## Important notes

N/A

## Packages installed

N/A

## Deployment note

N/A

## Related PRs branch | PR (branch_name) | (pr_link)

N/A

## Todos


## Screenshots

![screen recording 2019-01-28 at 02 40 pm](https://user-images.githubusercontent.com/31339738/51834349-137bbe80-230b-11e9-810c-44852139bff4.gif)
